app/api.py

```diff
@@ -75,6 +75,84 @@ def post_communityresource_register(body):
     return resource.to_dict(), 200
 
 
+def put_community_resource_edit(body):
+
+    geolocator = Nominatim() # will copy over from master branch
```

Comment:
The geopy module should be wrapped in some kind of service module as it requires you to make an http request to a 3rd party service every time you perform a lookup. Using geopy directly can be problematic in testing, we should have something similar to the manager classes that interact with the database.

Reply:
Does wrapping in a service module mean putting it in a separate private function (as suggested in code_review.txt)?  Or could it be a static function under the manager class?